src/modules/services/postgres.nix

```
@@ -40,8 +40,8 @@ let
               cat "${database.schema}" | postgres --single -E ${database.name}
             elif [ -d "${database.schema}" ]
             then
-              echo "Running sql files in ${database.schema}"
-              cat "${database.schema}/*.sql" | postgres --single -E ${database.name}
+              echo "Running sql files in ${database.schema}
+              cat ${database.schema}/*.sql | postgres --single -E ${database.name}
```

Another option, if you want the quotes, is:

```suggestion
              find -s "${database.schema}" -name "*.sql" -exec cat {} \; | postgres --single -E ${database.name}
```